### PR TITLE
Updates to WALKTHROUGH.md.

### PR DESCRIPTION
**Pull request by delbonis**
Updated to use the new address format, command responses (like `con`), and explanation for support cointypes that was added since it was originally written.

### CODE REVIEW

**Comment by delbonis** [comment]
I ended up making a new GitHub account anyways so someone can remove @treyzania from the org and add this once instead.

File: WALKTHROUGH.md [hunk 1]
```diff
@@ -100,22 +99,21 @@ Bob puts the pubkey@hostname for Alice and connects.  Then he says hi to Alice.
...
 
 Bob is connected to Alice and wants to open a payment channel. If he has enough (segwit) money and is connected to Alice, he can open a channel.
 
 ```
-lit-af# fund 1 50000000 0
+lit-af# fund 1 1 50000000 0
 ```
 
-This opens a channel with peer 1 (Alice) with a channel capacity of 50,000,000 satoshis (half a coin), and sends 0 satoshis over in the creation process.  Bob starts out with all 50,000,000 satoshis in the channel, so only he can send to Alice.
+This opens a channel with peer 1 (Alice) with a channel capacity of 50,000,000 satoshis (half a coin), and sends 0 satoshis over in the creation process.  Bob starts out with all 50,000,000 satoshis in the channel, so only he can send to Alice.  The second "1" is to specify the cointype, which is 1 for the Bitcoin Testnet3.
```
Comment (Varunram) [inline WALKTHROUGH.md]: s/for the/for should be better :)

Comment (delbonis) [reply WALKTHROUGH.md]: Fixed.

**Comment by adiabat** [comment]
Looks good, thanks for the update

**Pull request merged**